app/assets/stylesheets/partials/_navigation.css.scss

```diff
@@ -0,0 +1,18 @@
+.navbar {
+  @include clearfix;
+  border-bottom: 1px solid $base-border-color;
+  clear: both;
+  cursor: pointer;
+  line-height: 2;
+  position: relative;
```

Comment:
Properties should be ordered border-bottom, clear, cursor, line-height, margin, overflow, position, text-align, width